.azure/cve-pipeline.yaml

```diff
@@ -0,0 +1,89 @@
+# Triggers
+trigger: none
+pr: none
+
+# Parameters
+parameters:
+  - name: releaseVersion
+    displayName: Release Version
+    type: string
+  - name: useSuffix
+    displayName: Build suffixed images
+    type: boolean
+    default: true
+  - name: releaseSuffix
+    displayName: Release Suffix
+    type: number
+  - name: sourcePipelineId
+    displayName: Pipeline ID of the source build
+    type: number
+    default: 16
+    values:
+      - 16
```

Comment:
just out of curiosity - why 16?

Reply:
This is the ID of the pipeline the artifacts are taken from -> and that is in this case so far only the operators `build` pipeline which has ID 16. There might be more options in the future, but even if not, having it here helps to have it in one place.

Reply:
oh, I see, maybe we can add there comment with that explanation?

Reply:
I added the comments.